### DISCUSSION TITLE
It seems we can use poetry to install and deploy the project, why didn't mention it in wiki?

### DISCUSSION THREAD

**Discussion by JasonJarvan** [LazyAGI/LazyLLM#324] [General]
Hi community!
I found we have pyproject.toml and requreiments.txt in our project, and there is a check_requirements.py script to compare them.
I am confused why we have both, as both of them can be used as dependency, it's overlapped. 
And I guess we can support `poetry install` . If yes, why didn't mention it in wiki?
Or am I wrong?

**Comment by wzh1994** [comment]
When we build an official image, we will use pyproject.toml to lock the version of the dependent library to ensure that the images built on different systems are consistent; but when users actually use it, using poety to install will force the modification of the version of the locally installed library, such as pytorch, numpy, etc. Therefore, we provide an additional pip installation method to keep the locally installed packages from being updated when they meet the version requirements.
